src/main/java/bio/terra/cli/command/workspace/List.java

```diff
@@ -72,6 +72,7 @@ private enum Columns implements ColumnDefinition<UFWorkspaceLight> {
     ID("ID", w -> w.id, 40, LEFT),
     NAME("NAME", w -> w.name, 30, LEFT),
     GOOGLE_PROJECT("GOOGLE PROJECT", w -> w.googleProjectId, 30, LEFT),
+    AWS_ACCOUNT("AWS_ACCOUNT", w -> w.awsAccountId, 15, LEFT),
```

Comment:
`environmentAlias` might be more meaningful here (currently `saas` for our `devel` AWS Environent), or maybe a combination of `tenantAlias` + `environmentAlias` (`devel:saas` for `devel`)?

WDYT?

Reply:
AccountId  - likley to know which account their workspace is in.
EnvironmentAlias, tenantAlias - any use?
MajorVersion, OrganizationId - unlikely to be useful